ironfish-cli/src/multisigBroker/server.ts

```diff
@@ -223,6 +230,24 @@ export class MultisigServer {
     this.clients.delete(client.id)
   }
 
+  /**
+   * If a client has the given session ID and is connected, the associated
+   * session should still be considered active
+   */
+  private isSessionActive(sessionId: string): boolean {
+    for (const client of this.clients.values()) {
```

Comment:
Keeping this in mind for a followup task - do you think we should store a list of client IDs on the session? Here and in `broadcast` we iterate through all of the clients and filter out the ones that aren't in the right session 🫣

Reply:
Yeah I think that'd be a good change. I imagine we'll eventually get more and more logic around that, so having it easily accessible will be good.